cmd/protoc-gen-cpp-tableau-loader/main.go

```diff
@@ -17,10 +17,24 @@ const pbExt = "pb" // protobuf file extension
 var namespace *string
 var messagerSuffix *string
 
+// genMode can control only the `registry` code, or only generate the `loader` code
+// To avoid each change need a fully regenerated, for better dependency management
+var genMode *string
+
+const (
+	// normalMode All configuration protocols need to be entered at once to regenerate all code, the default option
+	normalMode = "normal"
+	// registryOnly Only the registry code is generated. Depend on all protocol files. If any protocol changes, the registry code need update
+	registryOnly = "registry_only"
+	// loaderOnly Only the loader code is generated. Could generate code only for protocols that change
+	loaderOnly = "loader_only"
+)
```

Comment:
Simplify and suggest more clear naming:
```
const (
	ModeDefault = "default" // generate all at once.
	ModeDelta   = "delta"   // only generate specified protos' corresponding "*.pc.h/cc" files and registry.pc.h/cc" files
)
```

Reply:
The `register_only` and `loader_only` modes have different inputs and outputs. They need to be used seperatly, and can't be simplified to `delta` mode.

About the comments, I think it will be better to clearly show the input and output for each mode.
But I can change the `normal` mode name to `default` if you would prefer, what do you think?

Reply:
ok, rename modes to be clear.